perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count,
+                    'workers')
```

Comment:
Relations in Juju generally handles configuration details between two services. Between something like mysql and wordpress, the relation between the two would trigger the creation of the database, users, etc. needed by wordpress on mysql's end, which would then send the appropriate credentials to the wordpress service.

In the case of cassandra-stress, the relation authorizes the vm(s) running cassandra-stress to access the cassandra cluster and does any preparations required for the two services to work together.